frontend/src/components/SkillGroup/SkillGroup.tsx

```diff
@@ -18,9 +19,11 @@ const SkillGroup: FC<ISkillGroupProps> = ({ skillFounded, skills }) => {
           </SkillsBox>
           <SkillsDivider />
         </SkillsGroupWrapper>
-      ))}
-    </>
-  );
-};
+      ))
+    ) : (
+      <NoContent message={NO_SKILLS} />
+    )}
+  </>
+);
 
-export default SkillGroup;
+export default memo(SkillGroup);
```

Comment:
Are you shure that it works correctly? skillFounded its array.  https://blog.bitsrc.io/optimize-your-react-app-with-react-memo-ec52447b09ba. 
![image](https://user-images.githubusercontent.com/61495158/173080227-431fe516-656f-4b48-a6cc-00e44d1774b0.png)

Reply:
need to discuss